ui/app/routes/clients.js

```diff
@@ -14,10 +13,14 @@ export default class ClientsRoute extends Route.extend(WithForbiddenState) {
     return this.get('system.leader');
   }
 
-  model() {
-    return RSVP.hash({
-      nodes: this.store.findAll('node'),
-      agents: this.store.findAll('agent'),
-    }).catch(notifyForbidden(this));
+  async model() {
+    try {
+      return {
+        nodes: await this.store.findAll('node'),
+        agents: await this.store.findAll('agent'),
+      };
```

Comment:
This is a change in behavior. Since this is using two `await`s, we're running these requests synchronously, not simultaneously.

I would still use RSVP's `hash` function for this (since you want both `findAll`s to run at the same time) and await on the hash, rather than each promise.

Also, it was there before your change, but it strikes me as odd that we're catching the error here and not in the [Route.error](https://guides.emberjs.com/release/routing/loading-and-error-substates/#toc_the-error-event). Might be a larger refactor than you're going for with just the `async/await`.

Reply:
That's such a good call out! We break pretty significantly from the Ember way on many things. Unsure if we're doing something funky with errors where we use the `notifyError` utility along with a `mixin` to update the controller to have an error. But that's a can of worms that I don't dare to open.